### DISCUSSION TITLE
How do I define Pygimli's DataContainer to store field measurement coordinates defined by easting, northing, and depth?"

### DISCUSSION THREAD

**Discussion by Chih-Hao-Kuo** [gimli-org/pyGIMLi#691] [Q&A]
[ERT_inverted_conductivity.csv](https://github.com/gimli-org/gimli/files/14850082/ERT_inverted_conductivity.csv)

**Comment by halbmy** [comment]
Your csv file is just a point list that can be nicely read by numpy
```
import numpy as np
DATA = np.genfromtxt("blabla.csv", names=True, delimiter=",")
print(DATA["longitude"])
```
You could also add selected fields to a data container
```
data = pg.DataContainer()
data["longitude"] = DATA["longitude"]
data["latitude"] = DATA["latitude"]
```
but with no huge benefit except things like `data.remove(data["latitude"] > 89)` or `data.subset(latitude=90)`. 

Our data container is made for index formats where you store positions only once and then you index it. This makes also spatial operations like `data.translate()` possible (upcoming).

From the filename I suspect this is an inversion result that is not at all related to a data container but rather a mesh.
